backend/btrixcloud/migrations/migration_0001_archives_to_orgs.py

```diff
@@ -0,0 +1,107 @@
+"""
+Migration 0001 - Archives to Orgs
+"""
+import os
+
+from pymongo.errors import InvalidName, OperationFailure
+
+
+class Migration:
+    """Migration class."""
+
+    COLLECTIONS_AID_TO_OID = [
+        "collections",
+        "crawl_configs",
+        "crawls",
+        "invites",
+        "profiles",
+    ]
+
+    MIGRATION_NAME = os.path.basename(__file__)
+
+    def __init__(self, mdb):
+        self.mdb = mdb
+
+    async def migrate_up_needed(self):
```

Comment:
I wonder if we should add a db version information somewhere, to make it clearer when migration is needed, and to which version?